pkg/concepts/targets.go

```diff
@@ -102,13 +104,23 @@ func renderJsonnetConcept(path string, avs *RenderValues, single bool) ([]File,
 	var bundle []File
 	out := make(manifest.List, 0, len(extract))
 	for _, m := range extract {
-		out = append(out, m)
 		bundle = append(bundle, File{
 			path:    fmt.Sprintf("%s_%s_%s.yaml", strings.ReplaceAll(m.APIVersion(), "/", "-"), m.Kind(), m.Metadata().Name()),
 			content: []byte(m.String()),
 		})
 	}
 
+	sort.Slice(bundle, func(i, j int) bool {
```

Comment:
we don't want to sort the `bundle` but the `out` list.

We can directly `sort.Slice(out, ... )` just the comparison function passed will compare the entries from bundle.

Reply:
please check my snippet here https://github.com/redradrat/kable/pull/24#pullrequestreview-799131358

Reply:
Thanks for reviewing. Just one sec. Will resubmit the change.

Reply:
> please check my snippet here [#24 (review)](https://github.com/redradrat/kable/pull/24#pullrequestreview-799131358)

Clear. Thanks. Pls see the latest changes.

Reply:
lgtm

that should produce the same functionality. Could you verify it?